lib/rules/require-curly-braces.js

```diff
@@ -72,7 +72,7 @@ module.exports.prototype = {
         if (typeIndex['case'] || typeIndex['default']) {
             file.iterateNodesByType('SwitchCase', function(node) {
                 // empty case statement
-                if (!node.consequent || node.consequent.length === 0) {
+                if (node.consequent.length === 0) {
```

Comment:
Parser API said: consequent should exists and be an Array in any cases. So no need to check it.